### DISCUSSION TITLE
Are there any plans of publishing this app in F-Droid or IzzyOnDroid?

### DISCUSSION THREAD

**Discussion by salazarbarrera** [visnkmr/backgroundappslist#170] [Ideas]
I guess that for any open source Android app, sooner or later there's someone who ends up asking this. Today, that someone it's me.

P.S.: Absolutely loving this app. Only one week and it is already an essential for me.

**Comment by visnkmr** [comment]
The app is not OSS, so currently we have no plans to publish the app to f droid or izzyondroid. We use github for tracking issues, discussions and for brainstorming. You can find our recent changes/commit history of the app [here](http://visnkmr.github.io/commits?reponame=12core1/bapl). You can find out a list of Open source apps on github profile page which includes netspeed monitor for PC, file manager written in rust that we are currently working on called filedime/iomer.

**Reply by salazarbarrera** [reply]
My bad, I saw the repo from mobile and didn't notice it didn't have the source code.
